index.js

```diff
@@ -29,67 +28,64 @@ DatabaseUpdates.prototype = Object.create(EventEmitter.prototype)
 
 DatabaseUpdates.prototype.getFiles = function getFiles() {
   try {
-    this.updateFiles = fs.readdirSync(this.updatePath)
+    this.updateFiles = fs
+      .readdirSync(this.updatePath)
       // exclude non-javascript files in the updates folder
-      .map(i => (path.extname(i) !== '.js' ? false : i))
+      .map((i) => (path.extname(i) !== '.js' ? false : i))
       // exclude falsy values and filenames that without a valid semver
-      .filter(i => i && semver.valid(i.split('-')[0]))
+      .filter((i) => i && semver.valid(i.split('-')[0]))
```

Comment:
```suggestion
      .filter((fn) => ['.js', '.ts'].includes(path.extname(fn)) && semver.valid(fn.split('-')[0]))
```

Reply:
I dont think there's any way for this module to reasonably accept TS files without bringing in typescript as a dependency to allow us to import TS code and call it dynamically, unfortunately.

Reply:
my suggestion was more to drop the map above and just use a filter. but I was just nit'ing.